templates/vars/vnet/ithc.json

```diff
@@ -3,7 +3,7 @@
   "contentVersion": "1.0.0.0",
   "parameters": {
     "hubVnetId": {
-      "value": "/subscriptions/fb084706-583f-4c9a-bdab-949aac66ba5c/resourceGroups/hmcts-hub-nonprodi/providers/Microsoft.Network/virtualNetworks/hmcts-hub-nonprodi"
```

Comment:
why this change?

Reply:
reverted, some reason thought ops-peerings covered this now.

Reply:
i think it only does ukwest, need to get rid of it from ops-peering but can do that when you move to terraform